createpythonpackage/main.py

```diff
@@ -101,6 +104,12 @@ def _create_package(args):
             }
         )
 
+    if not package_config.package_name.isidentifier():
+        raise CppError(
+            f"{package_config.package_name} is not a valid Python identifier. "
+            "Please consider using another package name."
```

Comment:
The user may be left wondering why their package name is not valid. Suggest adding

```
A valid identifier is the uppercase and lowercase letters A through Z, the underscore _ and, except for the first character, the digits 0 through 9.
```

Reply:
Sure, I think that can help. Is this the official description of what an identifier? We can just copy paste it to be sure we’re conveying the correct instructions.

Reply:
Yes it is with the exception PEP 3131 unicode support. https://docs.python.org/3/reference/lexical_analysis.html#identifiers